i3_resurrect/util.py

```diff
@@ -1,8 +1,27 @@
 import sys
-
+import string
 
 def eprint(*args, **kwargs):
     """
     Function for printing to stderr.
     """
     print(*args, file=sys.stderr, **kwargs)
+
+
+def filename_filter(filename):
+    """
+    Take a string and return a valid filename constructed from the string.
+"""
+    blacklist = '/\:*"<>|'
+    # Alternatively use whitelist approch
+    # whitelist = "-_.() %s%s" % (string.ascii_letters, string.digits)
```

Comment:
Not much point leaving these comments around. That whitelist wouldn't even allow for unicode characters anyway from the looks of it.